src/Illuminate/Database/Console/Migrations/RollbackCommand.php

```diff
@@ -57,10 +57,7 @@ public function handle()
 
         $this->migrator->usingConnection($this->option('database'), function () {
             $this->migrator->setOutput($this->output)->rollback(
-                $this->getMigrationPaths(), [
-                    'pretend' => $this->option('pretend'),
-                    'step' => (int) $this->option('step'),
-                ]
+                $this->getMigrationPaths(), $this->options()
```

Comment:
Since we now need the `--path` also, I think it's easier to send all the options, instead of rebuild an array with the same unmodified data.